src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -118,6 +120,18 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(this._configurationService.onDidChangeConfiguration(e => {
 			this.doubleClickEditEnabled = this._configurationService.getValue('notebook.enableDoubleClickEdit');
 		}));
+		this._register(DOM.addDisposableListener(window, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
```

Comment:
can we add the listener on the notebook cell element instead of window?

Reply:
@alanrenmsft I tried to add it to the notebook component but I cannot add listeners to divs. 
I'm not sure if we can add the listener to each notebook cell because we undo/redo when the notebook cell is not selected to prevent from triggering the undo/redo within the actual cell.

Reply:
Yeah - I think that this needs to go at the top level because we purposely run the logic only when nothing is active.

This goes back to my other comment though that generally I would expect keybindings to all be handled by the core command handling code instead of manually like this.